fedmsg_meta_fedora_infrastructure/conglomerators/copr/copr.py

```diff
@@ -0,0 +1,73 @@
+import fedmsg.meta.base
+from fedmsg_meta_fedora_infrastructure.fasshim import avatar_url
+
+
+class AbstractCoprConglomerator(fedmsg.meta.base.BaseConglomerator):
+    def can_handle(self, msg, **config):
+        return '.copr.' in msg['topic']
+
+    def merge(self, constituents, **config):
+        ms = constituents  # shorthand
+
+        agents = set([m['msg']['user'] for m in ms])
+        coprs = set([m['msg']['copr'] for m in ms])
+        starts = len([1 for m in ms
+                      if m['topic'].endswith('copr.build.start')])
+
+        rebuild_predicate = 'rebuilds' if starts > 1 else 'rebuild'
+        copr_predicate = 'coprs' if len(coprs) > 1 else 'copr'
+
+        agents = self.list_to_series(agents)
+        coprs = self.list_to_series(coprs)
+
+        subtitle = '{agents} kicked off {starts} {rebuild_predicate} ' + \
+            'of the {coprs} {copr_predicate}'
```

Comment:
Yeah, I thought ``foo kicked off 2 rebuilds of the bar copr`` sounded more natural.  Let's get a third opinion from someone else.